api/api/settings.py

```diff
@@ -141,6 +142,8 @@
 
 USE_TZ = True
 
+CORS_ORIGIN_ALLOW_ALL = True
+
```

Comment:
is this safe for production? 

my amateur knowledge of enabling CORS is you sometimes need to do it when making API calls on localhost as you get a CORS error, which you wouldn't normally get when deployed on a domain.

Reply:
https://stackoverflow.com/questions/39042799/cors-localhost-as-allowed-origin-in-production/39083377

Reply:
@calmacx, you are right that CORS are set up to allow API calls from other domains.

![image](https://user-images.githubusercontent.com/53081016/130627314-399c345e-6470-4db7-a272-ba8de4dfb7fe.png)

@spco Right now the CORS are set up to allow all domains, I have not tried it on production systems yet but I think they can be set up for a specific domain as shown in Figure.

Details of setting up CORS for Django App is available here: http://www.srikanthtechnologies.com/blog/python/enable_cors_for_django.aspx

Reply:
I think this needs further testing before we deploy any fix to prod, and I am minded to deploy this to prod as it stands. Am I right that we are still protected by the fact that all API calls require the secret token anyway, and we're not depending on cookies? But this would add further protection. Can you confirm my understanding is right?

Reply:
@spco Yes your understanding is correct that all API calls are protected by token authentication anyway.